### PR TITLE
fix image src

### LINKED ISSUE #786: Broken image reference in "Hello World Rust" template
On the [Hello World Rust](https://developers.cloudflare.com/workers/templates/pages/hello_world_rust) template, there's a broken image for the "Publish" step.

Screen Shot 2020-05-12 at 8 12 44 AM

The image src is `/workers/workers/templates/media/publish.gif` and should probably be `/workers/templates/media/publish.gif`.

### PR DESCRIPTION
**Pull request by victoriabernard92**
Screen Shot 2020-05-14 at 1 54 56 PM

fixes #786 
improve comment and var naming

### CODE REVIEW

**Comment by ispivey** [comment]
🤔  regex-based URL manipulation scares me 😂 

Why do we need to do this? Is it to ensure that authors can write `/workers/templates/media/publish.gif` or `/templates/media/publish.gif` and they both resolve to the same image?

**Comment by victoriabernard92** [comment]
@ispivey precisely. eventually i'd love to come up with a better practice for how we link within the `.md` files (are ref links the way to go? or /workers/* ? ) but I stuck to whatever conventions the hugo docs already used. I don't like regexing like this either but couldn't come up with anything better that worked and this has worked for a while for the other links

**Comment by ispivey** [comment]
👍  sorry for the slow turnaround time on the review. 

I think we should figure out a way to remove string replacement logic, rather than adding more, even if it requires changing and verifying links in the existing content. This feels very fragile! 

In particular, there are three different places I know of where we modify path strings using regex:
* The Link component you're editing here
* [workers-docs/workers-site/src/main.js](https://github.com/cloudflare/workers-docs/blob/temp-image/workers-docs/workers-site/src/main.js#L32-L62)
* [workers-docs/gatsby-node.js](https://github.com/cloudflare/workers-docs/blob/temp-image/workers-docs/gatsby-node.js#L50-L64)

It's hard to reason about what we expect to happen! I think this is the source of some weirdness, like #797 , and the fact that refreshing a page like https://developers.cloudflare.com/workers/tutorials will 301 you to `https://developers.cloudflare.com/workers/tutorials/` which will 200 and then update history to modify the URL to `https://developers.cloudflare.com/workers/tutorials` again.

So, two questions and suggestions:
1. Can we simply resolve #786 by updating the img src in `hello_world_rust.md` , to remove the `/workers` prefix, rather than adding more conditional logic to the Link component? 
1. Would you like to split out a separate RFC for simplifying the URL parsing/rewriting logic across the three components above, in order to address #797 and other associated complexity? I'm happy to discuss it more, and I'm sure @signalnerve would have suggestions too.

**Comment by victoriabernard92** [comment]
1. Yes! This does not change any conditional logic. It simply updates the comment and variable name for readability if you look at it. I can remove that but it seemed helpful
2. Agree. This is a big issue Kristian and I worked through with prefixes. It's not ideal but it works. Definitely deserving of a RFC

**Comment by signalnerve** [comment]
i'm good with this PR, but IMO we shouldn't have to do a ton of url rewriting to get the tutorials to render. i've been writing them in the format`/tutorials/<tutorial>/<asset>` and that seems to work, is that stuff getting picked up by some of these rewriters or is that the "best case" of how this stuff should be written? 

i don't know if i have a super strong opinion or suggestion on how to do it well, but i'm inclined to suggest that we just do whatever requires the least amount of custom URL rewriting, even if it means some manual work now: i agree with @ispivey that this kind of stuff is fragile and we've already gone through enough URL issues w/ the path/asset prefix work!

File: cloudflare-docs-theme/src/components/Link.tsx [hunk 1]
```diff
@@ -10,10 +10,10 @@ type LinkProps = {
   [x: string]: any // To improve types, might want to inherit from  GatsbyLinkProps<TState> instead
 }
 
-// replace anything with /workers or / prepended to just /workers or whatever is in publicPath
-const stripWorkers = (url: string, path: string) => {
-  const regex = new RegExp('^(\\/' + path + '){1,3}')
-  return url.replace(regex, '').replace(/^\/(?!\/)/, '/' + path + '/')
+// replace anything with /workers* or /* prepended to just /workers* or /publicPath*
```
Comment (victoriabernard92) [inline cloudflare-docs-theme/src/components/Link.tsx:13]: This is just changing the comment and renaming a variable for clarity no logic changed

**Pull request merged**